app/workers/reset_project_counters_worker.rb

```diff
@@ -0,0 +1,39 @@
+class ResetProjectCountersWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :data_high
+
+  def perform(project_id)
+    project = Project.find(project_id)
+
+    return unless project.launch_date
+
+    project.classifications_count = project.classifications.where("created_at >= ?", project.launch_date).count
+    project.classifiers_count = UserProjectPreference.connection.execute(<<-SQL)[0]["count"]
+      SELECT COUNT(DISTINCT(upp.user_id)) count FROM user_project_preferences upp
+      INNER JOIN classifications c ON c.user_id = upp.user_id
+      INNER JOIN projects p ON p.id = upp.project_id
+      WHERE upp.project_id = #{project.id} AND c.project_id = #{project.id} AND c.created_at >= p.launch_date;
+    SQL
+    project.save!
```

Comment:
Fine by me. That's why I did the save-bang method, so it at least wouldn't go unnoticed, but a surgical replacement of just the two values and ignoring validations is fine too.